nextmv/input.py

```diff
@@ -273,7 +273,7 @@ def _load_archive(
         csv_ext = ".csv"
         for file in os.listdir(dir_path):
             if file.endswith(csv_ext):
-                stripped = file.strip(csv_ext[1:]).strip(".")  # Python 3.8 forces this, instead of using removesuffix()
+                stripped = file.removesuffix(csv_ext)
```

Comment:
Most important change is this one

Reply:
Something like this would also have fixed it:

```python
if file.endswith(csv_ext):
    file = file[: -len(csv_ext)]
```

However, I am good with moving forward if we don't see issues with that among our users. As this PR already shows there are some other benefits. :relaxed: 

One question: Do we need to also allow `.CSV` or even be case-insensitive? :thinking: 

Probably not. Just some thoughts. Nothing blocking.

Reply:
Expecting "case-insensitivity" among files is probably a Windows only issue. :sweat_smile:

Reply:
Didn't thought about it that much. Thank you. I can adjust if it ever comes to that.